### DISCUSSION TITLE
Virus Detected

### DISCUSSION THREAD

**Discussion by smith581** [SanGraphic/QuickBoost#9] [General]
[](url
![VirusTotal Results](https://user-images.githubusercontent.com/10013955/178795424-80c39cb9-cc2b-47ad-a6fb-be62e0318693.PNG)
)

I always check my files using Virus Total before executing. This file shows it has 2 detections of virus. Please fix these.

**Comment by SanGraphic** [comment]
its a false positive, you can build the program from source and it would still give you the same false positive.
